src/main/java/ua/ithillel/lesson15/ProductMoving.java

```diff
@@ -0,0 +1,88 @@
+package ua.ithillel.lesson15;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class ProductMoving {
+
+    private final List<Product> list = new ArrayList<>();
+
+    public void add(Product product){
+        list.add(product);
+    }
+
+    public List<Product> getWholeList(){
+        return list;
+    }
+
+    public List<Product> getCategoryForPrice(Category category, double minPrice) {
```

Comment:
```suggestion
    public List<Product> getProductsByCategoryAndPrice(Category category, double minPrice) {
```
`getCategoryForPrice` - я це читаю, дай категорію для такої ціни. А по факту повертає не категорію, а список продуктів. 

Інші варіанти: `filterByCategoryAndPrice`, `getProductsForCategoryAndPrice`